storybook/stories/API/props/StoryParameters.ts

```diff
@@ -0,0 +1,6 @@
+import type { Parameters } from '@storybook/react';
+
+export const SHARED_STORY_PARAMS: Parameters = {
+  // Sets a delay of 1550ms before each story to give them time to animate (recharts animation default is 1500ms)
+  chromatic: { delay: 1550 },
+};
```

Comment:
Should these not be https://storybook.js.org/docs/react/writing-stories/parameters#global-parameters ?

Reply:
Lol 🙃 I was looking really hard for that but I couldn't find it. I think this works and is (obviously) easier to maintain. Thanks!